projects/open-telemetry/otel-gov-review.md

```diff
@@ -0,0 +1,504 @@
+OTel lists their [SIGs and Working Groups] in the README within the community
+repo which is generated from [sigs.yaml].
+
+OTel originally tracked their subprojects/repos and ownership in a
+[private admin repo], however they have opened a [PR to update sigs.yaml] in
+their community repo and have created a workflow that will keep it updated in
+the future.
+
+SIGs and subprojects follow the project level guidelines for membership,
+contributing etc, but MAY define additional criteria or roles to meet the
+needs of the subproject e.g. [opentelemetry-collector].
+
+
+[opentelementry-collector]: https://github.com/open-telemetry/opentelemetry-collector/blob/8951ae8caefcc33fd1767f304701237ce0bcf4b6/CONTRIBUTING.md#membership-roles-and-responsibilities
+
+### Contributors and Community
+
+**Contributor ladder with multiple roles for contributors.**
+<br>
+**Incubating:** Suggested | **Graduated:** Suggested
+
+OTel's [contributor ladder] is well documented and easily discoverable from the
+community repo.
+
+[contributor ladder]: https://github.com/open-telemetry/community/blob/fe12ecb89802550b4d3daa9a30646157c540b391/guides/contributor/membership.md
+
+
+**Clearly defined and discoverable process to submit issues or changes.**
+<br>
+**Incubating:** Required | **Graduated:** Required
+
+From a random sampling of 5 repos, they all had issue templates for the general
+set of things like features, bugs, etc.
+
+For large-scale technical decisions, Otel has an [RFC process called OTEPs] or
+OpenTelemetry Specification. The process and steps itself are very well
+documented, but not as easily discoverable. There are no specific links or
+references in the community repo and contributor/maintainer guide except for
+a comment in the [guidelines around donating a repo].
+
+
+[guidelines around donating a repo]: https://github.com/open-telemetry/community/blob/706839937eb5886bb07da6bd9d63245839e3cc2f/guides/contributor/donations.md?plain=1#L46
+
+**Project must have, and document, at least one public communications channel
+for users and/or contributors.**
+<br>
+**Incubating:** Required | **Graduated:** Required
+
+OTel has all their public [communication channels] available in the
+community/repo.
+
+
+**List and document all project communication channels, including subprojects
+(mail list/slack/etc.). List any non-public communications channels and what
+their special purpose is.**
+<br>
+**Incubating:** Required | **Graduated:** Required
+
+OTel has all their public [communication channels] available in the
+community/repo, other lists and private channels are listed in the
+[list of assets] in the community repo.
+
+[list of assets]: https://github.com/open-telemetry/community/blob/073235c1e018bc658235e384ff2429ade74898eb/assets.md
+
+
+**Up-to-date public meeting schedulers and/or integration with CNCF calendar.**
+<br>
+**Incubating:** Required | **Graduated:** Required
+
+OTel lists their public meetings in the [communication channels] section of
```

Comment:
link?

Reply:
It's a reference link, here it is directly:
https://github.com/open-telemetry/community/blob/073235c1e018bc658235e384ff2429ade74898eb/README.md#special-interest-groups